charts/kafka/README.md

```diff
@@ -0,0 +1,502 @@
+pod/palisade-kafka-0                    1/1     Running   1          30m
+pod/palisade-kafka-1                    1/1     Running   0          28m
+pod/palisade-kafka-2                    1/1     Running   0          27m
+pod/palisade-kafka-testclient           1/1     Running   0          30m
+pod/palisade-zookeeper-0                1/1     Running   0          30m
+pod/palisade-zookeeper-1                1/1     Running   0          29m
+pod/palisade-zookeeper-2                1/1     Running   0          29m
+
+NAME                                  TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)                      AGE
+service/palisade-kafka                ClusterIP   10.97.189.45     <none>        9092/TCP                     30m
+service/palisade-kafka-headless       ClusterIP   None             <none>        9092/TCP                     30m
+service/palisade-zookeeper            ClusterIP   10.99.212.137    <none>        2181/TCP                     30m
+service/palisade-zookeeper-headless   ClusterIP   None             <none>        2181/TCP,3888/TCP,2888/TCP   30m
+
+NAME                                  READY   AGE
+statefulset.apps/palisade-kafka       3/3     30m
+statefulset.apps/palisade-zookeeper   3/3     30m
+```
+
+To verify that the kafka service is operational perform the following:
+
+```bash
+kubectl  exec -ti palisade-kafka-testclient --namespace=dev -- ./bin/kafka-topics.sh --zookeeper palisade-zookeeper:2181 --list
+```
+
+This should result in the topics being displayed, if no topics have been created then a null output would be been.
+
+Create a topic as follows:
+```bash
+kubectl  exec -ti palisade-kafka-testclient --namespace=dev -- ./bin/kafka-topics.sh --create --replication-factor 1 --partitions 1 --zookeeper palisade-zookeeper:2181 --topic topicName
+Created topic "topicName".
+```
+
+
+
+This is an implementation of Kafka StatefulSet found here:
+
+ * https://github.com/Yolean/kubernetes-kafka
+
+## Pre Requisites:
+
+* Kubernetes 1.3 with alpha APIs enabled and support for storage classes
+
+* PV support on underlying infrastructure
+
+* Requires at least `v2.0.0-beta.1` version of helm to support
+  dependency management with requirements.yaml
+
+## StatefulSet Details
+
+* https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/
+
+## StatefulSet Caveats
+
+* https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#limitations
+
+## Chart Details
+
+This chart will do the following:
+
+* Implement a dynamically scalable kafka cluster using Kubernetes StatefulSets
+
+* Implement a dynamically scalable zookeeper cluster as another Kubernetes StatefulSet required for the Kafka cluster above
+
+* Expose Kafka protocol endpoints via NodePort services (optional)
+
+### Installing the Chart
+
+To install the chart with the release name `my-kafka` in the default
+namespace:
+
+```
+$ helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
```

Comment:
This instructions are for the original kafka charts. Might be worth moving down the earlier description around deployment and update this 'Installing the chart' section